gherkin/targeting.feature

```diff
@@ -145,8 +145,20 @@ Feature: Targeting rules
       | non-string-variant-targeting-flag | 2     |
       | empty-targeting-flag              | 1     |
 
-  @flagdcontext
-  Scenario: Use Flagd provided context
+  @contextEnrichment
```

Comment:
@aepfli I thought this was a better name and more consistent with the wording we use in code and options.